README.md

```diff
@@ -104,39 +102,45 @@ Usage:
 
 ```sh
 BEAKER_vagrant_box_tree=$vagrant_boxes_dir \
-BEAKER_box__puppet="simpci/server-6.2.0-RC1.el7-CentOS-7.0.x86-64" \
+BEAKER_box__puppet="simpci/server-6.2.0-RC1.el7-CentOS-6.9.x86-64" \
```

Comment:
el7 does not match 6.9

Reply:
This was still sitting in an unsaved vim buffer :man_facepalming: ; fix pushed in d4a221f